quickwit-indexing/src/actors/indexing_pipeline.rs

```diff
@@ -550,23 +546,28 @@ pub struct IndexingPipelineParams {
 impl IndexingPipelineParams {
     pub async fn try_new(
         index_metadata: IndexMetadata,
-        indexer_config: IndexerConfig,
+        source: SourceConfig,
+        indexing_settings: IndexingSettings,
+        data_dir_path: PathBuf,
+        split_store_max_num_bytes: usize,
+        split_store_max_num_splits: usize,
         metastore: Arc<dyn Metastore>,
         storage: Arc<dyn Storage>,
     ) -> anyhow::Result<Self> {
         let doc_mapper = index_metadata.build_doc_mapper()?;
-        let indexing_directory_path = indexer_config.data_dir_path.join(&index_metadata.index_id);
+        let indexing_directory_path = data_dir_path
```

Comment:
The path for an indexing directory is now `{data_source_path}/{index_id}/{source_id}`.